app/models/developer.rb

```diff
@@ -72,6 +72,7 @@ class Developer < ApplicationRecord
   scope :available_first, -> { where.not(available_on: nil).order(:available_on) }
   scope :featured, -> { where("featured_at >= ?", FEATURE_LENGTH.ago).order(featured_at: :desc) }
   scope :newest_first, -> { order(created_at: :desc) }
+  scope :product_feature_notifications, -> { where(product_feature_notifications: true) }
```

Comment:
Last change: let's rename this scope and column to `product_announcement_notifications` to match the notification and mailer.

Reply:
Sure.. Updating and pushing changes now